Dockerfile

```
@@ -0,0 +1,22 @@
+FROM golang:1.19.3-alpine as builder
```

```suggestion
FROM golang:1.19.4-alpine as builder
```
https://github.com/golang/go/releases/tag/go1.19.4 ;P

---

:bow: https://github.com/golang/go/releases/tag/go1.19.5 :joy: